Makefile

```
@@ -401,3 +401,10 @@ show-img:
 
 show-container-engine:
 	@echo -n $(CONTAINER_ENGINE)
+
+.PHONY: deployment-script
+deployment-script: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
```

Description is bit confusing to me, compared to the `deploy` target.

```
$ make help
[...]
  deploy           Deploy controller to the K8s cluster specified in ~/.kube/config.
  deployment-script  Deploy controller to the K8s cluster specified in ~/.kube/config.
```

Target name too, as a script IMO means something that I can execute; here we'll be applying the manifest, per the official K8s glossary (https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-manifest).


Suggestion:
```suggestion
deployment-manifest: manifests kustomize ## Generate operator manifest in a file ready to be deployed.
```